pkg/assets/bindata.go

```diff
@@ -83533,7 +83532,7 @@ h2,h3{page-break-after:avoid}
 .glyphicon-menu-up:before{content:"\e260"}
 *,:after,:before{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}
 html{font-size:10px;-webkit-tap-highlight-color:transparent}
-body{font-size:13px;line-height:1.66666667;color:#333}
+body{font-family:"Open Sans",Helvetica,Arial,sans-serif;font-size:13px;line-height:1.66666667;color:#333}
```

Comment:
There are some bindata changes I wouldn't expect. Can you check that it's right?